kore/src/Kore/Equation/Equation.hs

```diff
@@ -82,6 +82,8 @@ import qualified SQL
 
 data Equation variable = Equation
     { requires :: !(Predicate variable)
+    , argument :: !(Predicate variable)
```

Comment:
Should this be a `Maybe`, like `antiLeft`?

Reply:
Yes, you're right, thanks for noticing this!